### ISSUE TITLE
How to increase the classification accuracy

### ISSUE THREAD

**Issue by itruonghai** [ultralytics/yolov5#1307]
My model predict the bounding box is quite good, but the label classification on **_test_** is terrible (also the loss cls is very low). How to prevent it?

**Comment by siyangxie** [comment]
Do you mean your model performs good on training but bad on testing? That's called overfitting. There're plenty ways to fix this problem depending on different scenarios.
